client/src/components/layout.tsx

```diff
@@ -2,33 +2,33 @@ import React, {ReactNode} from 'react';
 import {GridContainer, Grid} from '@trussworks/react-uswds';
 import J40Header from './J40Header';
 import J40Footer from './J40Footer';
-import J40Aside from '../components/J40Aside';
 import {URLFlagProvider} from '../contexts/FlagContext';
-// this has to be wrong
+import {Helmet} from 'react-helmet';
+import {useIntl} from 'gatsby-plugin-intl';
 
 interface ILayoutProps {
   children: ReactNode,
   location: Location
 }
 
 const Layout = ({children, location}: ILayoutProps) => {
-  const isWidthFullPage = location.pathname.match(/cejst\/?/);
-  const conditionalAside = isWidthFullPage ? <></> : <J40Aside/>;
-  const gridCssClass = isWidthFullPage ? ' desktop:grid-col-12' :
-    'desktop:grid-col-9';
+  const intl = useIntl();
 
   // @ts-ignore
   return (
     <URLFlagProvider location={location}>
+      <Helmet htmlAttributes={{lang: intl.locale}}>
+        <meta charSet="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+      </Helmet>
       <J40Header/>
       <GridContainer containerSize={'desktop-lg'}
         className={'j40-grid-container'}>
         <Grid row>
           <main id={'main-content'}
-            className={'usa-layout-docs j40-main-content ' + gridCssClass}>
+            className={'usa-layout-docs j40-main-content desktop:grid-col-12'}>
```

Comment:
you could always extend the class `desktop:grid-col-12` into the `usa-layout-docs` class name in scss

Reply:
Yeah, my scss foo is weak. I'd love to do a separate PR to just clean up this stuff and pair work on it. @NatHillardUSDS and @vim-usds might like be interested as well.

Is it ok if I leave it messy for this PR?